launchpad_reporting/config/sample.yaml

```diff
@@ -0,0 +1,101 @@
+criterias:
+    # implements criteria #4 from requirements
+    - name: sla-non-triaged
+      implementation: criterias.NonTriaged
+      short-text: non-triaged
+      config:
+          # Bug not triaged in that number of days is considered
+          # to be violating triage SLA
+          - name: threshold
+            default: 1
+            text: Threshould to consider bug non-triaged in time, in days
+
+      # implements criteria #3 from requirements
+    - name: sla-full-lifecycle
+      implementation: criterias.SLAFullLifecycle
+      short-text: lifetime
+      hint-text: Bug of severity {importance} {with_customer_found} must be fixed in {threshold} days
+      config:
+          # Pick bug if it is critical and wasn't fixed in that
+          # number of days
+          - name: critical_threshold
+            default: 10
+          # Pick bug if it is high and wasn't fixed in that
+          # number of days
+          - name: high_threshold
+            default: 40
+
+      # implements criteria #5 from requirements
+    - name: sla-confirmed-triaged
+      implementation: criterias.SLAConfirmedTriaged
+      short-text: conf/tri
+      hint-text: The bug is hanging in 'Confirmed/Triaged' status for more than {threshold} days
+      config:
+            - name: threshold
+              default: 5
+
+      # implements criteria #6 from requirements
+    - name: sla-in-progress
+      implementation: criterias.SLAInProgress
+      short-text: in-progress
+      hint-text: Bug of severity {importance} {with_customer_found} is hanging in 'In Progress' status for more than {threshold} days
+      config:
+            - name: critical_customer_found_threshold
+              default: 2
+            - name: critical_threshold
+              default: 3
+            - name: high_customer_found_threshold
+              default: 5
+            - name: high_threshold
+              default: 7
+            - name: others_threshold
+              default: 11
+
+      # implements criteria for current HCF report
+      # (bug is either critical or high)
+    - name: hcf-affecting
+      implementation: criterias.HCFAffecting
+
+reports:
+
+    - name: non-triaged-in-time
+      text: Non-Triaged in Time Bugs
+      parameter: project
+      criterias:
+            - name: sla-non-triaged
+      # do not implement frequent-polling in current scope
+      frequent-polling: true
+        
+    - name: sla-report
+      text: SLA Report
+      parameter: milestone
+      group-by: team
+      display-triggere-criterias: true
```

Comment:
That is a typo here, the parameter should be
display-trigger-criterias

can you change that in config and in implementation itself?